lib/presentation/view/search.dart

```diff
@@ -1,12 +1,459 @@
+                      ),
+                      const Text(
+                        menuTitle,
+                        style: TextStyle(
+                          color: Colors.white,
+                          fontFamily: AssetConstants.nothingFont,
+                          fontSize: Dimensions.bigFontSize,
+                          decoration: TextDecoration.underline,
+                        ),
+                      ),
+                      Positioned(
+                        top: menuCloseIconSize,
+                        left: menuCloseIconSize,
+                        child: GestureDetector(
+                          onTap: () {
+                            searchResultsController.resetSearchResults();
+                            filterController.resetFilters();
+                            Get.back();
+                          },
+                          child: const Icon(
+                            Icons.close,
+                            color: Colors.white,
+                          ),
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+                ListTile(
+                  title: const Text(cuisineTitle),
+                  subtitle: DropdownButton<String>(
+                    isExpanded: true,
+                    value: filterController.cuisine.value.propertyName,
+                    items: CuisineEnum.values
+                        .map(
+                          (cuisine) => DropdownMenuItem<String>(
+                            value: cuisine.propertyName,
+                            child: Text(cuisine.propertyName),
+                          ),
+                        )
+                        .toList(),
+                    onChanged: (value) {
+                      filterController.updateCuisine(
+                        CuisineEnum.values.firstWhere(
+                          (e) => e.propertyName == value,
+                        ),
+                      );
+                    },
+                  ),
+                ),
+                ListTile(
+                  title: const Text(mealTypeTitle),
+                  subtitle: DropdownButton<String>(
+                    isExpanded: true,
+                    value: filterController.mealType.value.propertyName,
+                    items: MealTypeEnum.values
+                        .map(
+                          (mealType) => DropdownMenuItem<String>(
+                            value: mealType.propertyName,
+                            child: Text(mealType.propertyName),
+                          ),
+                        )
+                        .toList(),
+                    onChanged: (value) {
+                      filterController.updateMealType(
+                        MealTypeEnum.values.firstWhere(
+                          (e) => e.propertyName == value,
+                        ),
+                      );
+                    },
```

Comment:
You can create a new widget and as parameter you can use value` (filterController.mealType.value.propertyName or filterController.cuisine.value.PropertyName)`
item list could be (MealTypeEnum and CuisineEnum) and also you can send onChanged function too

Reply:
Created a DropDownFilter widget.